tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_subvolgroup_octal_modes.py

```diff
@@ -0,0 +1,392 @@
+            format="%a",
+        )
+        stat_of_octal_mode_on_fuse_dir1 = fs_util.get_stats(
+            client=clients[0],
+            file_path=fuse_mounting_dir_1.rstrip("/") + subvolgroup1_getpath,
+            format="%a",
+        )
+        stat_of_octal_mode_on_fuse_dir2 = fs_util.get_stats(
+            client=clients[0],
+            file_path=fuse_mounting_dir_2.rstrip("/") + subvolgroup2_getpath,
+            format="%a",
+        )
+        stat_of_octal_mode_on_fuse_dir3 = fs_util.get_stats(
+            client=clients[1],
+            file_path=fuse_mounting_dir_3.rstrip("/") + subvolgroup_ec_getpath,
+            format="%a",
+        )
+
+        if int(subgroup_1_mode) != int(stat_of_octal_mode_on_kernel_dir1) and int(
+            subgroup_1_mode
+        ) != int(stat_of_octal_mode_on_fuse_dir1):
+            log.error("Octal values are mismatching on subvolgroup_1")
+            return 1
+        if int(subgroup_2_mode) != int(stat_of_octal_mode_on_kernel_dir2) and int(
+            subgroup_2_mode
+        ) != int(stat_of_octal_mode_on_fuse_dir2):
+            log.error("Octal values are mismatching on subvolgroup_2")
+            return 1
+        if int(subgroup_ec_mode) != int(stat_of_octal_mode_on_kernel_dir3) and int(
+            subgroup_ec_mode
+        ) != int(stat_of_octal_mode_on_fuse_dir3):
+            log.error("Octal values are mismatching on subvolgroup_ec1")
+            return 1
+
+        log.info("Run IO's")
+        with parallel() as p:
+            for i in [
+                kernel_mounting_dir_1,
+                fuse_mounting_dir_1,
+                kernel_mounting_dir_2,
+                fuse_mounting_dir_2,
+            ]:
+                p.spawn(fs_util.run_ios, clients[0], i)
+
+            for i in [kernel_mounting_dir_3, fuse_mounting_dir_3]:
+                p.spawn(fs_util.run_ios, clients[1], i)
+
+        log.info("Clean up the system")
+        for subvolume in subvolume_list:
+            fs_util.remove_subvolume(clients[0], **subvolume)
+
+        for subvolumegroup in subvolumegroup_list:
+            fs_util.remove_subvolumegroup(clients[0], **subvolumegroup)
+
+        for i in [kernel_mounting_dir_1, kernel_mounting_dir_2]:
+            fs_util.client_clean_up(
+                "umount", kernel_clients=[clients[0]], mounting_dir=i
+            )
+
+        for i in [fuse_mounting_dir_1, fuse_mounting_dir_2]:
+            fs_util.client_clean_up("umount", fuse_clients=[clients[0]], mounting_dir=i)
+
+        fs_util.client_clean_up(
+            "umount", kernel_clients=[clients[1]], mounting_dir=kernel_mounting_dir_3
+        )
+        fs_util.client_clean_up(
+            "umount", kernel_clients=[clients[1]], mounting_dir=fuse_mounting_dir_3
+        )
+
+        clients[1].exec_command(
+            sudo=True, cmd="ceph config set mon mon_allow_pool_delete true"
+        )
+
+        return 0
+    except Exception as e:
+        log.info(e)
```

Comment:
can you change it to log.error instead of log.info().

With this change we can see the error logs in reportportal directly.

May be in all the places where we have loggers in exception block